libs/libc/stdlib/lib_exit.c

```diff
@@ -46,22 +46,33 @@ FAR void *__dso_handle = &__dso_handle;
  * Public Functions
  ****************************************************************************/
 
-void exit(int status)
+void _Exit(int status)
```

Comment:
I think flushing the streams here is not correct:

The _Exit() and _exit() functions shall not call functions registered with atexit() nor any registered signal handlers. Open streams shall not be flushed.  Whether open streams are closed (without flushing) is implementation-defined. Finally, the calling process shall be terminated with the consequences described below.

https://pubs.opengroup.org/onlinepubs/9699919799/functions/_Exit.html

Reply:
You are right. the description in https://en.cppreference.com/w/c/program/_Exit isn't accuracy.